src/mockServices/mockApiResponses/getMemberProfile.js

```diff
@@ -22,16 +22,20 @@ export const getMemberProfile = {
   projects: [
     {
       id: '01932a8a-4690-4468-83f0-df8c269c47be',
-      name: 'Project Unicorn',
+      name: 'Project Unicorn UI',
       defaultPictureUrl:
         'https://projectunicorn.blob.core.windows.net/projects/Image1.png',
+      projectUrl: '/projects/project-unicorn-ui',
+      ownerProfileUrl: '/members/GreenWithMv',
       type: 'dev community app'
```

Comment:
Add a new property to the response so that we know who the owner of the project is
```suggestion
      type: 'dev community app',
      memberId: '01932a8a-4690-4468-83f0-df8c269c47be'
```